capa/features/extractors/viv/extractor.py

```diff
@@ -19,19 +20,25 @@
 import capa.features.extractors.viv.basicblock
 from capa.features.common import Feature
 from capa.features.address import Address, AbsoluteVirtualAddress
-from capa.features.extractors.base_extractor import BBHandle, InsnHandle, FunctionHandle, StaticFeatureExtractor
+from capa.features.extractors.base_extractor import BBHandle, InsnHandle, FunctionHandle, FeatureExtractor
 
 logger = logging.getLogger(__name__)
 
 
-class VivisectFeatureExtractor(StaticFeatureExtractor):
+class VivisectFeatureExtractor(FeatureExtractor):
     def __init__(self, vw, path, os):
         super().__init__()
         self.vw = vw
         self.path = path
         with open(self.path, "rb") as f:
             self.buf = f.read()
 
+        self.sample_hashes = (
+            hashlib.md5().update(self.buf).hexdigest(),
+            hashlib.sha1().update(self.buf).hexdigest(),
+            hashlib.sha256().update(self.buf).hexdigest(),
+        )
```

Comment:
lets introduce a dataclass to represent this tuple, so the order and names are easy to recognize as a human